src/internal/hooks/use-telemetry/index.ts

```diff
@@ -4,7 +4,7 @@ import { useComponentMetrics } from '@cloudscape-design/component-toolkit/intern
 import { PACKAGE_SOURCE, PACKAGE_VERSION, THEME } from '../../environment';
 import { useVisualRefresh } from '../use-visual-mode';
 
-export function useTelemetry(componentName: string) {
+export function useTelemetry(componentName: string, config?: Parameters<typeof useComponentMetrics>[2]) {
```

Comment:
Temporary using `typeof useComponentMetrics` reflection, it will switch to first class types after merging this: https://github.com/cloudscape-design/component-toolkit/pull/51

Reply:
Updated now